recipes/gfflu/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "gfflu" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace("_", "-") }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 49a4df62c2538b59f404df6b6f12afa7d261dea6269ba9998ecfed24874c79da
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --ignore-installed -vv .
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - hatch
+  run:
+    - blast
+    - miniprot
+    - python >=3.8
+    - pip
```

Comment:
pretty sure you dont need `pip` in the `run` section

Reply:
Thanks! You're right. Removed in 5c7f1d8067